integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -124,6 +126,16 @@ def bulk_assessment_level_export(self):
         # Create a record of each subsection from every enterprise enrollment
         for enterprise_enrollment in enrollment_queryset:
             if not LearnerExporter.has_data_sharing_consent(enterprise_enrollment):
+                # Adding logging to debug the issue we are having with a customer using SAPSF channel
+                if self.enterprise_customer.uuid == SAPSF_CUST_UUID:
```

Comment:
You can remove this check `if self.enterprise_customer.uuid == SAPSF_CUST_UUID:`. We can filter this on the basis of `enterprise_customer.uuid` value in Splunk.